docs/Configuration.md

```diff
@@ -376,3 +381,83 @@ class RootComponent extends React.Component {
 
 AppRegistry.registerComponent('MyApp', () => RootComponent);
 ```
+
+### External (pre-built) bundles
+
+Check out [monorepo fixture](../fixtures/monorepo) for a complete an example.
+
+To use external bundles, first built the bundles that will be considered pre-built. You only need a single bundle config:
+```js
+import { withPolyfills, makeConfig } from '@haul-bundler/preset-0.60';
+import { join } from 'path';
+
+const entry = withPolyfills([
+  require.resolve('react'),
+  require.resolve('react-native'),
+  require.resolve('react-navigation'),
+]);
+
+export default makeConfig({
+  bundles: {
+    base_dll: ({ dev }) => ({
+      entry,
+      dll: true,
+      // Packager server will use development bundle, so it should be built as a plain JS bundle.
+      type: dev ? 'basic-bundle' : 'indexed-ram-bundle',
+    }),
+  },
+});
+```
+
+Use `--skip-host-check` option when building pre-built bundles:
```

Comment:
Is it possible to automatically skip host check for pre-built bundles? This flag seems quite unintuitive

Reply:
Not sure how to reliably detect when to run this check. IMO we either leave this flag or remove the check.

Reply:
If the check may cover some of the common fail points, let's leave it as is